src/Grid.stories.js

```diff
@@ -0,0 +1,128 @@
+/** @jsx jsx */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+// import { css, cx } from 'react-emotion';
+import { css, jsx } from '@emotion/core';
+
+import {
+  GridContainer, Grid, Row, Column, Col,
+} from './Grid';
+
+const containerStyle = css`
+  border: 1px solid black;
+  background-color: #eee;
+`;
+
+const rowStyle = css`
+  border-top: 1px solid black;
+
+  &:first-child {
+    border-top: none;
+  }
+
+  &:nth-child(odd) {
+    background-color: #ddd;
+  }
+`;
+
+const body = css`
+  font-family: sans-serif;
+  font-size: 16px;
+  font-weight: lighter;
+`;
+
+storiesOf('Emotion CSS Grid', module)
+  .add('Default', () => (
```

Comment:
This seems like a pretty basic, limited-use example of the grid. Could we work in some additional stories with advanced use cases? Images, buttons, forms, etc.?

Doesn't need to be anything styled like our components - the default browser styles on some markup would be fine. But it'd be good to validate the grid against some fancier content.